sources/packages/api/src/services/institution/institution.service.ts

```diff
@@ -537,4 +539,48 @@ export class InstitutionService extends RecordDataModelService<Institution> {
       .where("institution.id = :institutionId", { institutionId })
       .getOne();
   }
+
+  /**
+   * Service to get notes for a student.
+   * @param institutionId
+   * @param noteType
+   * @returns Notes
+   */
+  async getInstitutionNotes(
+    institutionId: number,
+    noteType?: NoteType,
+  ): Promise<Note[]> {
+    const institutionNoteQuery = this.repo
+      .createQueryBuilder("institution")
+      .select([
+        "institution.id",
+        "note.noteType",
+        "note.description",
+        "note.createdAt",
+        "user.firstName",
+        "user.lastName",
+      ])
+      .innerJoin("institution.notes", "note")
+      .innerJoin("note.creator", "user")
+      .where("institution.id = :institutionId", { institutionId });
+    if (noteType) {
+      institutionNoteQuery.andWhere("note.noteType = :noteType", { noteType });
+    }
+
+    const institution = await institutionNoteQuery.getOne();
```

Comment:
why line `571`, you can directly return this right

Reply:
Changed as discussed